scss/overrides/_buttons.scss

```diff
@@ -28,3 +28,23 @@
     }
   }
 }
+
+a.btn {
```

Comment:
Wrong file. These aren't overrides for Foundation site style guide elements.

Reply:
Is `.btn-tag` a modified version on top of `.btn-secondary`?

Reply:
oops, i'll move that to a different file then. nope, `.btn-tag` is the new capsule-like button that Nat is trying to add to the style guide.